astropy/units/core.py

```diff
@@ -1090,12 +1090,37 @@ def get_err_str(unit):
 
         raise UnitConversionError(f"{unit_str} and {other_str} are not convertible")
 
-    def _get_converter(self, other, equivalencies=[]):
-        """Get a converter for values in ``self`` to ``other``.
+    def get_converter(self, other, equivalencies=[]):
+        """
+        Create a function that converts values from this unit to another.
+
+        Parameters
+        ----------
+        other : unit-like
+            The unit to convert to.
+        equivalencies : list of equivalence pairs, optional
```

Comment:
readthedocs stumbles on this (in the three places where the doc is used). In other places where the function is public, I see that it does `list of tuple` in the definition (and then `list of equivalence pairs` in the explanation, as you have here).